js/exercise-01.js

```diff
@@ -0,0 +1,26 @@
+/* 
+    1. Create a variable called 'name' that points at a string containing your name
+    2. Define function 'updateValue', which receives a value and updates variable 'name'
+    3. On the next line, call function 'updateValue' with your favourite musician's name
+      Log 'name' value.
+    4. On the next line, call function 'updateValue' with value 'true'
+      Log 'name' value.
+    5. On the next line, call function 'updateValue' value '28'
+      Log 'name' value.
+
+*/
+
+const name = "Alan Castillo";
```

Comment:
This step is not necessary. You can directly define a variable using `let`

```suggestion
let name = "Alan Castillo";
```

FIX: remove `const` definition and set using only `let`.